src/clients/authorization/auth-evaluate.css

```
@@ -20,4 +20,19 @@ button#reevaluate-btn {
   margin-right: var(--pf-global--spacer--md);
 }
 
+button#authorization-eval, button#authorization-revert {
+  margin-right: 16px;
+}
+
+.kc-identity-information, .kc-permissions {
+  border: none !important;
+}
 
+.kc-identity-information > div.pf-c-card__header.kc-form-panel__header {
+  padding-top: 0px;
```

You can omit the unit here. Perhaps it would be better to use a utility class though.

```suggestion
  padding-top: 0;
```